test/Microsoft.AspNetCore.Razor.Test/TestFiles/CodeGenerator/Output/NestedScriptTagTagHelpers.cs

```diff
@@ -57,16 +57,16 @@ public NestedScriptTagTagHelpers()
                 __tagHelperExecutionContext.Add(__TestNamespace_InputTagHelper);
                 __TestNamespace_InputTagHelper2 = CreateTagHelper<global::TestNamespace.InputTagHelper2>();
                 __tagHelperExecutionContext.Add(__TestNamespace_InputTagHelper2);
-                StartTagHelperWritingScope(null);
+                BeginWriteTagHelperAttribute();
                 WriteLiteral("2000 + ");
 #line 8 "NestedScriptTagTagHelpers.cshtml"
                                             Write(ViewBag.DefaultInterval);
 
 #line default
 #line hidden
                 WriteLiteral(" + 1");
-                __tagHelperStringValueBuffer = EndTagHelperWritingScope();
-                __tagHelperExecutionContext.AddHtmlAttribute("data-interval", Html.Raw(__tagHelperStringValueBuffer.GetContent(HtmlEncoder)));
+                __tagHelperStringValueBuffer = EndWriteTagHelperAttribute();
+                __tagHelperExecutionContext.AddHtmlAttribute("data-interval", Html.Raw(__tagHelperStringValueBuffer));
```

Comment:
Q: Do `HtmlString`s like these (`Html.Raw`) have a significant impact in the profile?